packages/cosmos_ui_components/lib/components/template/cosmos_adaptive_action_sheet.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:adaptive_action_sheet/adaptive_action_sheet.dart';
+import 'package:flutter/material.dart';
+
+Future showCosmosAdaptiveActionSheet(
+  BuildContext context, {
+  required String title,
+  required List<CosmosActionSheetItem> actions,
+  String cancelText = 'Cancel',
+}) async =>
+    showAdaptiveActionSheet(
+      context: context,
+      title: Text(title),
+      actions: actions
+          .map(
+            (e) => BottomSheetAction(
+              title: Text(e.text, style: TextStyle(color: e.isCriticalAction ? Colors.red : Colors.blue)),
```

Comment:
we should use colors defined in `CosmosTheme`, feel free to introduce `actionSheetDestructive`. for positive leave the color unchanged, the library will use the color default for given platform

Reply:
Perfect. There is one issue. The design shows color blue for non-destructive color whereas the library gives black color as default

Reply:
for iOS it should be lightBlue, no? That's what the library internal code shows

Reply:
Nope, it shows as black when I give no color:
![Screenshot 2021-11-25 at 9 56 07 PM](https://user-images.githubusercontent.com/25613623/143479579-90ae91b1-1c2e-48fc-a4cd-246e6c14c1fb.png)
.

Reply:
try changing it to this:
```suggestion
              title: Text(e.text, style: e.isCriticalAction ? TextStyle(color: Colors.red) : null),
```

this will make the action lightBlue by default, if its not destructive

Reply:
Just did that. Nothing changes. The code vs the result below
![Screenshot 2021-11-25 at 10 06 26 PM](https://user-images.githubusercontent.com/25613623/143480665-e0881578-8869-4608-b9cd-9c2132e67989.png)